components/ui-api-layer/docs/usage-kind-resources.md

```diff
@@ -0,0 +1,12 @@
+# Query usageKindResources
+
+The query returns all instances of the resources specified by **usageKind** in the given **environment**. 
```

Comment:
``The query returns all instances of the resources specified by the **usageKind** parameter in the given **environment**.``